usagov/vote-gov-drupal
# Vote-2230: Updates from W3 markup tests

## Jira ticket

[Vote-2230](https://cm-jira.usa.gov/browse/VOTE-2230)

## Description

Minor markup edits found my usa.gov team... I also used the same tool and checked the other pages, however there were errors that I couldn't found/reproduce or it is flagging something that is needed to be used.   

## Deployment and testing

### Post-deploy steps

1. run `lando retune`

### QA/Testing instructions

1. Inspect mark up and verify that it is no longer present and review code removed and ensure that was nothing taken out that is needed 

## Checklist for the Developer




## Checklist for the Peer Reviewers
